docs/csharp/programming-guide/xmldoc/processing-the-xml-file.md

```diff
@@ -72,7 +72,7 @@ The compiler generates an ID string for each construct in your code that is tagg
   
 -   For generic types, the name of the type will be followed by a back tick and then a number that indicates the number of generic type parameters.  For example,  
   
-     `<member name="T:SampleClass`2">` is the tag for a type that is defined as `public class SampleClass\<T, U>`.  
+     ``<member name="T:SampleClass`2">`` is the tag for a type that is defined as `public class SampleClass\<T, U>`.  
```

Comment:
@yvt that's a good catch. Not related to your PR, but in the same sentence: we can remove the backslash in the second inline code block.